assets/src/edit-story/components/library/karma/libraryTabs.karma.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../karma';
+
+describe('LibraryTabs integration', () => {
+  let fixture;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    await fixture.render();
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  describe('keyboad navigation', () => {
+    beforeEach(async () => {
+      const textTab = fixture.container.querySelector('#library-tab-media');
+      await fixture.events.focus(textTab);
+    });
+
+    it('should be on the media tab', async () => {
+      await fixture.waitOnScreen(
+        fixture.container.querySelector('#library-pane-media')
+      );
+      await fixture.snapshot();
+    });
+
+    fit('should switch tabs on left and right keys', async () => {
+      await fixture.events.keyboard.press('ArrowRight');
+
+      // @todo: what's the best way to confirm switching of a tab?
```

Comment:
You can check the pane's `aria-expanded` attribute:

```js
      const textPane = fixture.container.querySelector('#library-pane-text');
      expect(textPane.getAttribute('aria-expanded')).toBe(true);
```

Reply:
Or that `hidden` is not present or false on the pane, but I prefer the other method for this one.

The `hidden` check could however be made to be a generic matcher such as `expect(textPane).not.toBeHidden();`.

Reply:
I see. I do like the idea of `aria-expanded`. Though there could be a bug where aria-expanded is set, but we forgot to show it. That being said, `toBeHidden` is also good. However, some implementation could keep elements off screen instead of "hidden". What's the safest from integration point of view?

Reply:
Added `aria-expanded` test, but also kept `waitOnScreen`. IMHO that one is still less ambiguous than `toBeHidden`.